ds-prep-python-tuples/main.py

```diff
@@ -0,0 +1,31 @@
+passenger = (12, True, "Bonnell, Miss. Elizabeth", "female", 58)
+print(passenger)
+name = passenger[2]
+print(name)
+age = passenger[-1]
+print(age)
+print(name, age)
+survived_and_name = passenger[1:3]
+print(survived_and_name)
+gender_and_age = passenger[3:5]
```

Comment:
For getting `gender_and_age`, please only specify the start of the range:

---

![image](https://user-images.githubusercontent.com/16143771/149249692-0439477f-7aa9-4167-9bba-407e8057d161.png)

Reply:
Tim from what I understand we are supposed to output gender and age so ('female', 58) is incorrect? I am not sure how to approach this problem unless I utilize a negative indice?

Reply:
![image](https://user-images.githubusercontent.com/54218913/149251352-6edd9bb4-41cf-4985-8fe3-faa5dae40cfb.png)